spring-cloud-kubernetes-leader/pom.xml

```diff
@@ -37,6 +37,16 @@
 			<groupId>org.springframework.integration</groupId>
 			<artifactId>spring-integration-core</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-actuator</artifactId>
+			<optional>true</optional>
+		</dependency>
+		<dependency>
+			<groupId>io.rest-assured</groupId>
+			<artifactId>rest-assured</artifactId>
```

Comment:
I'd rather not use rest assured here. `WebTestClient` should be sufficient.

Reply:
I also updated the other tests I'd based mine one.